traffic_ops/testing/api/v5/origins_test.go

```diff
@@ -630,12 +630,14 @@ func GetOriginID(t *testing.T, name string) func() int {
 		assert.RequireNoError(t, err, "Get Origins Request failed with error:", err)
 		assert.RequireEqual(t, 1, len(origins.Response), "Expected response object length 1, but got %d", len(origins.Response))
 		assert.RequireNotNil(t, origins.Response[0].ID, "Expected ID to not be nil.")
-		return *origins.Response[0].ID
+		return origins.Response[0].ID
 	}
 }
 
 func CreateTestOrigins(t *testing.T) {
 	for _, origin := range testData.Origins {
+		origin.TenantID = GetTenantID(t, "root")()
+		origin.DeliveryServiceID = GetDeliveryServiceId(t, origin.DeliveryService)()
```

Comment:
Do you know why you had to add this? It's because the changes you made to the `originIDs` function in the client aren't working, because you're looking for `nil` where it's impossible for `nil` to occur.

Reply:
here i have added gettenantID and getdeliveryserviceID as testdata is not having tenant id and dsID so with the name am trying to add the missing data. to testdata